### PR TITLE
Updating MLP-OP to DSPEC 2.0

### PR DESCRIPTION
**Pull request by matdenkas**
Updating MLP-OP dspec to 2.0. 

To check this PR... its gonna suck. I wrote a checklist you can find here:
https://docs.google.com/spreadsheets/d/1eV9fRGkZnrJYPFxOCErO9p_oEVL2Mu1whJM-uyYW-30/edit?usp=sharing

Besides that its just comparing the old and new version determining no mistakes were made in the update. Sorry 😭

### CODE REVIEW

**Review by lovelysandlonelys — APPROVED** [review]
looks good to me and ran on my machine

File: data/dspec/MLP-OP.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+    "dspecVersion": "2.0",
     "modelName": "MLP-OP",
     "modelVersion": "2.0.0",
     "author": "Christian Duff",
```
Comment (lovelysandlonelys) [inline data/dspec/MLP-OP.json:5]: in the other two requests you have ./filename but in this one you just have `"modelFileName": "MLP-OP",`

Comment (matdenkas) [reply data/dspec/MLP-OP.json:5]: Both our valid paths, ./is more explicit. We use a pathing library so it understands what ./ means. Either is fine, im not sure which way would be better

Comment (lovelysandlonelys) [reply data/dspec/MLP-OP.json:5]: i think that we have ./in most of the other files so maybe that just for consistency?

File: data/dspec/MLP-OP.json [hunk 2]
```diff
@@ -16,36 +17,51 @@
...
             "_name": "Water Temp",
             "location": "NPSBI",
             "source": "LIGHTHOUSE",
             "series": "dWaterTmp",
             "unit": "celsius",
-            "type": "float",
             "interval": 3600,
-            "range": [0, -23]
+            "range": [0, -23],
+            "outKey": "NPSBI_DWL_24"
```
[the remark this answers is not in the document]
Comment (matdenkas) [reply data/dspec/MLP-OP.json:29]: Fixed!

**Pull request merged**